cloud_governance/tag_cluster/tag_cluster_resouces.py

```diff
@@ -376,6 +376,8 @@ def cluster_instance(self):
             cluster_process.join()
             non_cluster_process.join()
             return ids.get()
+        else:
+            return []
```

Comment:
Why we return empty list?

Reply:
when we don't have any data, it doesn't return any thing.